src/DynamoCore/Library/LibraryServices.cs

```diff
@@ -1090,14 +1094,16 @@ public static class Categories
 
         public class LibraryLoadFailedEventArgs : EventArgs
```

Comment:
please add summaries for properties and class.

Reply:
Summaries have been added